packages/client/src/pages/Game.tsx

```diff
@@ -87,6 +94,9 @@ export const Game = ({ gameState, gameScore }: { gameState: GameState; gameScore
       });
     }
   };
+
+  const isActive = !isSpectator && activePlayer === player?.username;
```

Comment:
do you need this logic? I don't expect to have an activePlayer that is not a player username right?

Reply:
I think you need some sort of check yeah... Before it was `activePlayer === player?.username`. If we make both opponents as you said above, no, but otherwise we need something I think.

Reply:
why do you need extra logic, activePlayer will always come from server right? It can't possibily be a spectator isn't it?

Reply:
The game state is the same for all, it's not user-specific. So, it's the client's responsibility to process it according to its needs. In this case, `activePlayer` is just the one playing now, doesn't know anything about the current user viewing the browser. In the same way, we don't emit a flag `isPlaying`, to a specific socket, to tell that that player is the active one, we just have an `activePlayer` and then depending on the username we know if it's the user's turn.

Reply:
I see now ok it can be weird otherwise. Let's go with this and later on we change it when we refactor player / opponents components then